mesonbuild/compilers/c.py

```diff
@@ -1284,9 +1288,13 @@ def __init__(self, exelist, version, compiler_type, is_cross, exe_wrapper=None,
 
     def get_options(self):
         opts = CCompiler.get_options(self)
+        c_stds = ['c89', 'c99', 'c11']
+        g_stds = ['gnu89', 'gnu99', 'gnu11']
+        if version_compare(self.version, '>=8.0.0'):
+            c_stds += ['c17','c18']
+            g_stds += ['gnu17','gnu18']
```

Comment:
__[Flake8]__ [E231] missing whitespace after ',' ([link](https://sider.review/gh/repos/19784232/pulls/5062#issue-3829497))

<sub>posted by Sider</sub>

Reply:
Just sent a commit to fix, will now try to squash it.